### PR TITLE
fix(demo): fix link to hosted dropdown component

### LINKED ISSUE #737: [BUG] Documentation - Broken link to hosted dropdown
# 🐞 Bug report

### Description

The link to `hosted dropdown` in [dropdown directive page](https://taiga-ui.dev/directives/dropdown) leads nowhere.

### PR DESCRIPTION
**Pull request by hakimio**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0-beta.4/)
-   [ ] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

-   [x] Bugfix
-   [ ] Feature
-   [ ] Refactoring (no functional changes, no api changes)
-   [ ] Other... Please describe:

## What is the current behavior?

Closes #737

## Does this PR introduce a breaking change?

-   [ ] Yes
-   [x] No

### CODE REVIEW

File: projects/demo/src/modules/directives/dropdown/dropdown.template.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
         <p i18n>
             See also
-            <a tuiLink [routerLink]="['/directives/hosted-dropdown']"
+            <a tuiLink [routerLink]="['/components/hosted-dropdown']"
```
Comment (splincode) [inline projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: @nsbarsukov e2e tests don't cover this case?

Comment (waterplea) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: They don't go over links on pages :)

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/directives/dropdown/dropdown.template.html (hunk 1 above)
Comment (hakimio) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: @splincode where exactly do you want me to update the test?

**Comment by hakimio** [comment]
Build failure seems to be unrelated.

File: projects/demo/src/modules/directives/dropdown/dropdown.template.html (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: @hakimio can you write a simple e2e test
@waterplea if needed? ))

Comment (hakimio) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: Yes, I can. Should it be in `demo-integrations/cypress/tests/demo/dropdown` or somewhere else?
@waterplea do you want to have the test?

Comment (waterplea) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: I don't think this is necessary

Comment (nsbarsukov) [reply projects/demo/src/modules/directives/dropdown/dropdown.template.html:11]: > @nsbarsukov e2e tests don't cover this case?

I think that covering the `demo` app with tests is redundant. The main thing for us is to cover our components (which we publish as the library) with tests.

**Pull request merged**